.gitignore

```diff
@@ -56,3 +56,5 @@ target/
 *~
 *.swp
 site/
+
+.idea
```

Comment:
Well, I use IDEA so that keeps me from having to have to see git complain about non-added files. Would be nice if you could leave it. But if not, I will survive ;-)